### PR TITLE
Footer redesign

### PR DESCRIPTION
**Pull request by bohdan-ukr**

### CODE REVIEW

File: src/pages/home/style.scss [hunk 1]
```diff
@@ -317,6 +317,10 @@
   padding: 1.5rem 25vh;
 }
 
+.error_input {
+  color: red;
```
Comment (AndriyTykhan) [inline src/pages/home/style.scss:321]: Let just use tailwind

File: src/components/footer/footer.js [hunk 2]
```diff
@@ -1,27 +1,53 @@
+import logo from "assets/logo-modal.svg";
+import logoSmall from "assets/logo-small.svg";
+import ContactUs from "components/footer/contact-us";
 import React from "react";
 import { useTranslation } from "react-i18next";
-import ContactUs from "components/footer/contact-us";
-import { LogoIcon } from "icons";
-
+import { Link } from "react-router-dom";
```
Comment (AndriyTykhan) [inline src/components/footer/footer.js:6]: Keep import order correct

File: src/components/footer/footer.js [hunk 3]
```diff
@@ -1,38 +1,66 @@
...
+              )
+          )}
+        </div>
+        <div className="footer__about-us">
+          <p className="footer__text">{t('FOOTER.CONTACT_US')}</p>
+          <ContactUs
+            emailAdress="mailto:immortall.lv@gmail.com"
+            emailName="immortall.lv@gmail.com"
+            phoneNumber="tel:+380730297300"
+            phoneNumberText="+380730297300"
```
Comment (AndriyTykhan) [inline src/components/footer/footer.js:58]: Please move this props to constants

File: src/components/footer/footer.js [hunk 4]
```diff
@@ -1,38 +1,66 @@
+import logo from "assets/logo-modal.svg";
+import logoSmall from "assets/logo-small.svg";
+import ContactUs from "components/footer/contact-us";
 import React from "react";
```
Comment (AndriyTykhan) [inline src/components/footer/footer.js:4]: Keep import order!

And person who left comment, resolves it

**Review by AndriyTykhan — CHANGES_REQUESTED** [review]
Fix comments

File: src/components/footer/footer.js (hunk 4 above)
Comment (bohdan-ukr) [reply src/components/footer/footer.js:4]: Sorry, Its my configuration in vs code setting and every time when im saving anything it sometimes change order. I will resolve it

**Review by AndriyTykhan — APPROVED** [review]

**Pull request merged**